blazar/plugins/devices/device_plugin.py

```diff
@@ -465,6 +465,7 @@ def list_allocations(self, query, detail=False):
         options['detail'] = detail
         devices_allocations = self.query_device_allocations(devices_id_list,
                                                             **options)
+        self.add_extra_allocation_info(devices_allocations)
```

Comment:
Just wondering, what if `extras` was a GET parameters? That might be more flexible. In this arrangement, an interesting config value would be to support disabling some extra fields from being requested? Maybe a future enhancement. We don't currently have anything sensitive in Blazar.